### PR TITLE
[ci]: Disable slow integration tests

### PR DESCRIPTION
**Pull request by 6r1d**

### CODE REVIEW

**Review by SamHSmith — APPROVED** [review]
LGTM, but aleksandr might care about the missing newline at the end of `data_model/derive/Cargo.toml`.

File: client/tests/integration/roles.rs [hunk 1]
```diff
@@ -12,6 +12,7 @@ use iroha_permissions_validators::public_blockchain::{
 };
 use test_network::*;
 
+#[ignore]
```
Comment (appetrosyan) [inline client/tests/integration/roles.rs:15]: Missed a spot

File: client/tests/integration/events/pipeline.rs [hunk 2]
```diff
@@ -8,22 +8,26 @@ use test_network::*;
 
 use super::Configuration;
 
+#[allow(dead_code)]
```
Comment (appetrosyan) [inline client/tests/integration/events/pipeline.rs:11]: So the dead code shouldn't be removed, given that it should also be possible to re-enable these tests without re-compiling.

**Review by appetrosyan — DISMISSED** [review]

**Comment by appetrosyan** [comment]
> LGTM, but aleksandr might care about the missing newline at the end of data_model/derive/Cargo.toml.

Actually, newline at the end of file is an anti-pattern that I don't much care for.

File: client/tests/integration/roles.rs (hunk 1 above)
Comment (appetrosyan) [reply client/tests/integration/roles.rs:15]:
```suggestion
#[ignore = "ignore, more in #2851"]
```

File: client/tests/integration/events/pipeline.rs (hunk 2 above)
Comment (appetrosyan) [reply client/tests/integration/events/pipeline.rs:11]:
```suggestion
// Needed to re-enable ignored tests. 
#[allow(dead_code)]
```

**Pull request merged**